### PR TITLE
Clarifies that IAP user access is non operator

### PR DESCRIPTION
**Pull request by dorbin**

### CODE REVIEW

**Comment by dorbin** [comment]
@duftler 
This is in response to a user's (internal) friction log, which I will send you separately.

**Comment by duftler** [comment]
Thanks. I see the comment there, but I still don't think this change makes it clearer overall. Both the script and the linked UI can be used to grant non-operator, IAP user access.

I think he should have used the script, but didn't realize that it does what he was looking to do. Just adding a comment to the manual, alternative path doesn't resolve this.

File: scripts/manage/landing_page_secured.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ~/spinnaker-for-gcp/scripts/manage/grant_iap_access.sh
 ```
 
-Alternatively, you can manually grant the `IAP-secured Web App User` role on the `spinnaker/spin-deck` resource to the user you'd like to grant access to [here](https://console.developers.google.com/security/iap?project={{project-id}}).
+To give a user non-operator access to Spinnaker, grant the `IAP-secured Web App User` role on the `spinnaker/spin-deck` resource to the user you'd like to grant access to [here](https://console.developers.google.com/security/iap?project={{project-id}}).
```
Comment (demobox) [inline scripts/manage/landing_page_secured.md]:
```suggestion
To give a user access to Spinnaker without making them an administrator, grant the `IAP-secured Web App User` role on the `spinnaker/spin-deck` resource to the user you'd like to grant access to [here](https://console.developers.google.com/security/iap?project={{project-id}}).
```

Comment (demobox) [reply scripts/manage/landing_page_secured.md]: I'm not sure if "non-operator access" will make sense to readers immediately?

**Comment by dorbin** [comment]
Fair enough.
Based on @duftler 's feedback, I'm closing this PR.

**Pull request closed without merging**